health/health.d/bcache.conf

```diff
@@ -9,9 +9,9 @@ component: Disk
     every: 1m
      warn: $this > 0
     delay: up 2m down 1h multiplier 1.5 max 2h
-     info: number of times data was read from the cache, \
-           the bucket was reused and invalidated in the last 10 minutes \
-           (when this occurs the data is reread from the backing device)
+     info: Number of bcache read races in the last minute. \
```

Comment:
@ilyam8 we could elaborate on what Bcache is and add links https://en.wikipedia.org/wiki/Bcache

Reply:
The details and links will be in the Bcache charts descriptions and alarms guides.